reference/5.1/Microsoft.PowerShell.Management/Push-Location.md

```diff
@@ -157,8 +157,8 @@ To make a location stack the current location stack, use the StackName parameter
 `Set-Location` cmdlet. For more information about location stacks, see the [Notes](#notes).
 
 > [!NOTE]
-> `Push-Location` cannot add a location to the unnamed default stack unless it is the current
-> location stack.
+`Push-Location` cannot add a location to the unnamed default stack unless it is the current
+location stack.
```

Comment:
This is a bug in PlatyPS that we need to fix. I don't want to change how we author NOTE blocks when we are in the process of fixing PlatyPS. Please revert this change.

Reply:
Agree.
Good to hear that PlatyPS in being fixed.